src/geometries/plane.js

```diff
@@ -0,0 +1,13 @@
+var registerGeometry = require('../core/geometry').registerGeometry;
+var THREE = require('../lib/three');
+
+registerGeometry('plane', {
+  schema: {
+    height: {default: 1, min: 0},
+    width: {default: 1, min: 0}
+  },
+
+  init: function (data) {
+    this.geometry = new THREE.PlaneGeometry(data.width, data.height);
```

Comment:
I am going to make every geometry a buffer geometry by default, then you specify `buffer: false` if you want otherwise. But that's done with `THREE.BufferGeometry().fromGeometry( geometry, settings );` so we can't hardcode PlaneBufferGeometry here.